model/deprecated/network.yaml

```diff
@@ -8,19 +8,20 @@ groups:
       - id: sock.peer.name
         type: string
         stability: deprecated
-        brief: Deprecated, use `server.socket.domain` on client spans.
+        brief: Deprecated, since this was intended to be used for proxies on client spans, use `proxy.address`.
         examples: ['/var/my.sock']
       - id: sock.peer.addr
         type: string
         stability: deprecated
         brief: >
-          Deprecated, use `server.socket.address` on client spans and `client.socket.address` on server spans.
+          Deprecated, if this doesn't represent a proxy address, use `server.ip` on client spans and `client.ip` on server spans.
+          If this represents a proxy address, use `proxy.address`.
         examples: ['192.168.0.1']
       - id: sock.peer.port
         type: int
         stability: deprecated
         examples: [65531]
-        brief: Deprecated, use `server.socket.port` on client spans and `client.socket.port` on server spans.
+        brief: Deprecated, since this was intended to be used for proxies, use `proxy.port`.
```

Comment:
I'm not sure if this is right?

Reply:
I'd change it to something like "use `proxy.port` on client side when connecting through forward proxy and `server.port` otherwise"